po/R-es.po

```diff
@@ -239,21 +239,23 @@ msgstr "Primeros pasos"
 msgid "More articles..."
 msgstr "Más artículos..."
 
+# Assuming feminine (e.g. functions). Arguments in Spanish are masculine (e.g. obsoleto)
 #: package.R:300
 msgid "deprecated"
-msgstr ""
+msgstr "obsoleta"
 
+# Assuming feminine (e.g. functions). Arguments in Spanish are masculine (e.g. reemplazado)
```

Comment:
Luckily these are only used for functions currently (in a badge). If we extend to arguments too, we'll need to figure out how to handle properly.